spark-indexer/src/main/java/org/uniprot/store/spark/indexer/taxonomy/reader/TaxonomyDeletedRDDReader.java

```diff
@@ -30,13 +30,15 @@ public JavaRDD<TaxonomyDocument> load() {
     private Dataset<Row> loadNodeRow() {
         JavaSparkContext sparkContext = jobParameter.getSparkContext();
         Config applicationConfig = jobParameter.getApplicationConfig();
+        String taxDb = jobParameter.getTaxDb();
+        boolean isReadDb = READ.equals(taxDb);
```

Comment:
Suggestion of logic:

Here you could set databasePropertyPrefix = "database."+jobParameter.getTaxDb().toLowerCase();

And below you will not need the 3 if clauses. 
.option("url",  applicationConfig.getString(databasePropertyPrefix+".url"))

Reply:
Do you think we could extract this logic to a Utils method like getDatabaseConnectionOptions? because we use it in many places. It could return the DataFrameReader option. this way you would only need to append the database name outside this util class, or pass it as parameter.